packages/swapper/src/swappers/cow/CowSwapper.ts

```diff
@@ -69,13 +70,25 @@ export class CowSwapper implements Swapper {
   }
 
   filterBuyAssetsBySellAssetId(args: BuyAssetBySellIdInput): AssetId[] {
-    console.info(args)
-    return []
+    const { assetIds = [], sellAssetId } = args
+    if (
+      !sellAssetId?.startsWith('eip155:1/erc20') ||
```

Comment:
Is there a way to ask cowswap what assets it supports?

We had to do it this way for zrx because theres no good way to get all the assets it supports from an api.

But if cowswap has an api with all supported assets it would be better to load those up in the initialize() function and do it how we do with thorchain swapper

Reply:
I did not find a way to fetch supported assets but will look again

Reply:
@elmutt unfortunately it seems there is no api / no easy way to get supported assets for cowswap